common/lib/xmodule/xmodule/seq_module.py

```diff
@@ -179,7 +179,9 @@ def student_view(self, context):
             rendered_child = child.render(STUDENT_VIEW, context)
             fragment.add_frag_resources(rendered_child)
 
-            titles = child.get_content_titles()
+            # `titles` is a list of titles to inject into the sequential tooltip display.
+            # We omit any blank titles to avoid blank lines in the tooltip display.
+            titles = [title for title in child.get_content_titles() if len(title) > 0]
```

Comment:
Nitpick: it might be better to do something like this:
```python
titles = [title.strip() for title in child.get_content_titles() if title.strip()]
```
This way, you're stripping out any leading and trailing whitespace in the titles. Any title that is *just* whitespace will end up as the empty string, which is falsy in Python and so will not pass the conditional.